packages/ag-charts-community/src/chart/interaction/animationManager.ts

```diff
@@ -148,6 +149,25 @@ export class AnimationManager extends BaseManager<AnimationEventType, AnimationE
         }
     }
 
+    public stopByAnimationId(id: string) {
+        try {
+            if (id != null && this.controllers.has(id)) {
+                this.controllers.get(id)?.stop();
+            }
+        } catch (error: unknown) {
+            this.failsafeOnError(error);
+            return;
+        }
+    }
+
+    public stopByAnimationIdPrefix(prefix: string) {
```

Comment:
![image](https://github.com/ag-grid/ag-charts/assets/9638588/f9e00533-03da-4fff-855f-6e64e3718038)

Reply:
Replaced with `Animation.groupId` and a `AnimationManager.stopAnimationByGroupId()`.